src/Mod/PartDesign/App/FeaturePrimitive.cpp

```diff
@@ -56,10 +56,13 @@ using namespace PartDesign;
 
 namespace PartDesign {
 
-const App::PropertyQuantityConstraint::Constraints torusRangeV = {-180.0,180.0,1.0};
-const App::PropertyQuantityConstraint::Constraints angleRangeU = {0.0,360.0,1.0};
-const App::PropertyQuantityConstraint::Constraints angleRangeV = {-90.0,90.0,1.0};
-const App::PropertyQuantityConstraint::Constraints quantityRange  = {0.0,FLT_MAX,0.1};
+const App::PropertyQuantityConstraint::Constraints torusRangeV = { -180.0, 180.0, 1.0 };
+const App::PropertyQuantityConstraint::Constraints angleRangeU = { 0.0, 360.0, 1.0 };
+const App::PropertyQuantityConstraint::Constraints angleRangeV = { -90.0, 90.0, 1.0 };
+// it turned out that OCC cannot e.g. create a box with a width of Precision::Confusion()
+// with double Precision::Confusion() all geometrixc promitives can be created

```

Comment:
A couple typos, and a suggested word change (since the C++ programmer in me read the word "double" to mean something else entirely :smile: .
```suggestion
// with twice Precision::Confusion() all geometric primitives can be created
```

Reply:
thanks, fixed